src/Filesystem/DirectoryReader.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Framework3\Filesystem;
+
+class DirectoryReader
+{
+    /**
+     * @param string $dir
+     *
+     * @return array|false
+     */
+    public function scan(string $dir)
+    {
+        if (!is_dir($dir)) {
```

Comment:
Codacy found an issue: [The use of function is_dir() is discouraged](https://app.codacy.com/gh/pierregaimard/framework3/pullRequest?prid=6279572)